recipes/quickjs/all/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 3.4)
+project(quickjs C)
+
+option(USE_BIGNUM "Use bignum" ON)
+option(DUMP_LEAKS "dump leaks" OFF)
+
+include_directories(${QUICKJS_SRC_DIR})
```

Comment:
I'd recommend using `target_include_directories()` instead of `include_directories()`.

Reply:
I agree with you!
Fixed.